example/src/main.rs

```diff
@@ -7,23 +7,22 @@ extern crate ethereum_types as types;
 extern crate rustc_hex;
 extern crate solaris;
 
-
 fn main() {
-    solaris::main(include_bytes!("../res/BadgeReg_sol_BadgeReg.abi"));
```

Comment:
Why is it reformatted? I think we should stick to 4 spaces and rustfmt for this repo. Will add an `.editorconfig`

Reply:
i forgot that this repo uses spaces since the rest of parity uses tabs. will undo the reformatting